apps/kol-amelamdim/pages/register/index.tsx

```diff
@@ -18,69 +18,80 @@ const Register = () => {
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
   const [error, setError] = useState<string>('');
+  const [loading, setLoading] = useState<boolean>(false);
 
   const handleSubmit = async (e) => {
     e.preventDefault();
     if (validator.isEmail(email)) {
       try {
+        setLoading(true);
         const { data } = await axios.post('/api/register', {
           email,
           password,
         });
         if (data.success) {
-          router.push('/');
+          await router.push('/');
+          setLoading(false);
         }
       } catch (error) {
+        setLoading(false);
         setError(error.response.data.message.heb);
       }
     } else {
+      setLoading(false);
       setError(API_ERRORS.invalidEmailError.message.heb);
     }
   };
 
   return (
     <StyledPageContainer>
-      <form onSubmit={handleSubmit}>
-        <Grid container direction={'column'}>
-          <Typography variant="h3" component="h2" sx={{ mt: 2 }}>
-            הרשמה
-          </Typography>
-          <TextField
-            sx={{ mt: 2 }}
-            required
-            id="outlined-required"
-            label="אימייל"
-            value={email}
-            onChange={(e) => setEmail(e.target.value)}
-            error={!!error}
-          />
-          <TextField
-            sx={{ mt: 2 }}
-            value={password}
-            label="סיסמא"
-            type="password"
-            required
-            autoComplete="current-password"
-            onChange={(e) => setPassword(e.target.value)}
-            error={!!error}
-          />
-          <Button
-            sx={{ mt: 2 }}
-            variant="contained"
-            type="submit"
-            disabled={!email || !password}
-          >
-            הרשם
-          </Button>
-        </Grid>
-      </form>
-      <Grid container sx={{ mt: 2 }}>
-        <Typography component="h4">כבר נרשמתם?&nbsp;</Typography>
-        <NextLink href="/login" passHref>
-          <MUILink>לחצו להתחברות</MUILink>
-        </NextLink>
-      </Grid>
-      {error && <FormError>{error}</FormError>}
+      {loading ? (
+        <div>טוען....</div>
```

Comment:
This div have no styling?

Reply:
For now i didnt make it, this is in my todolist